### DISCUSSION TITLE
servidor git para usar local para windows

### DISCUSSION THREAD

**Discussion by leandroruel** [frontendbr/forum#286] [Perguntas e Respostas]
gostaria de saber se vocês conhecem um servidor de git para windows, pode ser opensource, para deploy. estamos usando uma alternativa aqui bem engessada.

**Comment by willycamargo** [comment]
Qual a necessidade da empresa? Realmente precisa ser uma alternativa local? Não podem usar serviços como github, bitbucket, gitlab, etc?

Desculpa, eu não conheço nenhuma alternativa local, só estou comentando mesmo para tentar dar uma alternativa ao que procura =)

**Comment by lazarofl** [comment]
@leandroruel, você pode usar o [GitHub Enterprise](https://github.com/pricing), [Bitbucket Server](https://www.atlassian.com/software/bitbucket/server), [GitLab](https://about.gitlab.com/), [Microsoft Team Foundation Server](https://www.visualstudio.com/products/tfs-overview-vs), dentre outros, mas, de fato, assim como o @willycamargo comentou, seria legal entender a necessidade da sua empresa. Acho bem difícil hoje em dia se falar em manter servidores on premise para esse tipo de serviço aos desenvolvedores pois os serviços na núvem são cada vez mais comuns e seguros. Por sinal, todos esses que sitei tem o serviço principal deles 100% online.

Espero ter ajudado.

**Comment by leandroruel** [comment]
@willycamargo tem que ser local, na intranet da empresa mesmo, motivo: porque o boss quer. =p

**Comment by leandroruel** [comment]
@lazarofl nós só temos windows server aqui, mas futuramente, depois de terminar um projeto de api aqui, vamos montar um servidor linux e usar uma alternativa do gitlab ou bitbucket

**Comment by ericktatsui** [comment]
Cara, para um servidor windows local eu to usando o Bonobo Git Server.
Ele é simples mas faz o que promete.

**Comment by brunoravanhani** [comment]
Esse Bonobo parece legal

**Comment by felipefialho** [comment]
Resolveu @leandroruel ?

**Comment by leandroruel** [comment]
não no windows né, mas convecemos o pessoal de infra que vamos instalar uma VM para rodar o gitlab. pode fechar seu moderador
